CHANGELOG.rst

```diff
@@ -10,6 +10,11 @@ and this project adheres to `Semantic Versioning <http://semver.org/`_.
 Unreleased
 ==========
 
+v0.8.0 - 2022-02-22
+===================
+- Updated to use py-buzz 3.0
+- Changed to use pyproject-flake8 for linting
```

Comment:
Why?

Reply:
Because all of the other quality check tools we use are configurable in pyproject.toml. Vanilla flake8 is not. pyproject-flake8 is a thin wrapper that reads config from pyproject.toml. This will allow us to keep the quality check tool configuration in one place.